go.mod

```diff
@@ -18,6 +20,8 @@ require (
 require (
 	github.com/agnivade/levenshtein v1.1.0 // indirect
 	github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/form3tech-oss/jwt-go v3.2.5+incompatible // indirect
```

Comment:
Check: How come you used these jwt libraries? On jwt.io you can see list of libraries that are nice to use: https://jwt.io/

Reply:
those libs are used indirectly. I think dgrijalva/jwt-go is an old version of golang-jwt/jwt that along with MicahParks/keyfunc.

But yes, I should add tests... Wanna help 😉 ?

Reply:
Yeah I'd like to help if I just knew how to write them :D

Reply:
Easy to learn 😆